.github/workflows/test-updatecli-run.yml

```
@@ -0,0 +1,27 @@
+name: test-updatecli-run
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/test-updatecli-run.yml'
+      - 'updatecli/run/**'
+  push:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/test-updatecli-run.yml'
+      - 'updatecli/run/**'
+
+permissions:
+  contents: read
+
+jobs:
+  test-updatecli-run:
```

```suggestion
  test:
```

This way, we can have a single required status check named `test`.

See status checks in: https://github.com/elastic/oblt-actions/pull/6